lib/internal/fs/glob.js

```diff
@@ -178,6 +178,8 @@ class Glob {
   constructor(pattern, options = kEmptyObject) {
     validateObject(options, 'options');
     const { exclude, cwd } = options;
+    const overrides = options.overrides || kEmptyObject;
+    validateObject(overrides, 'options.overrides');
```

Comment:
No need to call `validateObject` even if it's not defined.
```suggestion
    const overrides = kEmptyObject;
    if (options.overrides != null) {
      validateObject(options.overrides, 'options.overrides');
      overrides = options.overrides;
    }  
```